packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,230 @@
+import * as Sentry from '@sentry/browser';
+import { EventProcessor, Hub, Integration, Scope } from '@sentry/types';
+
+/**
+ * NodeJS integration
+ *
+ * Provides a mechanism for NodeJS
+ * that raises an exception for handled, unhandled, timeout and similar times of error
+ * and captures the same in Sentry Dashboard
+ */
+export class AWSLambda implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'AWSLambda';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = AWSLambda.id;
+
+  /**
+   * context.
+   */
+  private _awsContext: {
+    getRemainingTimeInMillis: () => number;
+    callbackWaitsForEmptyEventLoop: boolean;
+    awsRequestId: string;
+    functionName: string;
+    functionVersion: string;
+    invokedFunctionArn: string;
+    logGroupName: string;
+    logStreamName: string;
+  };
+
+  /**
+   * timeout flag.
+   */
+  private _timeoutWarning?: boolean = false;
+
+  /**
+   * @inheritDoc
+   */
+
+  /**
+   * flush time in milliseconds to set time for flush.
+   */
+  private _flushTime?: number;
+
+  public constructor(options: { context?: any; timeoutWarning?: boolean; flushTime?: number } = {}) {
```

Comment:
Please use `flushTimeout` name instead. This is what we already use in node handlers.

Reply:
made this change in the new pull request.